calico_containers/pycalico/block.py

```diff
@@ -179,18 +200,20 @@ def auto_assign(self, num, handle_id, attributes, host,
         :param attributes: Contents of this dict will be stored with the
         assignment and can be queried using get_assignment_attributes().  Must
         be JSON serializable.
-        :param host: The host ID to use for affinity in when assigning IP
-        addresses.
+        :param host: The ID of the host requesting addresses.
         :param affinity_check: If true, verify that this block's affinity
-        matches the supplied host and throw a NoHostAffinityWarning if it
-        doesn't.  Set to false to disable this check.
+        matches the supplied host and throw a NoHostAffinityError if it
```

Comment:
Is ignoring it the correct behavior? Or should we be raising an error indicating that the calling code is doing something wrong?